9.2/root/usr/share/container-scripts/postgresql/README.md

```diff
@@ -33,6 +33,9 @@ You can also set the following mount points by passing the `-v /host:/container`
 | :----------------------- | ------------------------------------- |
 |  `/var/lib/pgsql/data`   | PostgreSQL database cluster directory |
 
+
+At location `${CONTAINER_SCRIPTS_PATH}/post-init.sh` should be located script which defines all post initialization tasks. By default there is no script and only way how to get script into location is using this image as base image. In this case you should do `COPY path-to-your-script ${CONTAINER_SCRIPTS_PATH}/post-init.sh`.
```

Comment:
I suggest describing this way of mounting the file into the container, so that people don't need to build their own images:

https://docs.docker.com/engine/userguide/containers/dockervolumes/#mount-a-host-file-as-a-data-volume